src/H5Dpkg.h

```diff
@@ -43,7 +44,7 @@
 
 /* Set the minimum object header size to create objects with */
 #define H5D_MINHDR_SIZE 256
-
+#if 0
```

Comment:
Is the below macro still used? If not, I'd be tempted to just remove it rather than ifdef it out

Reply:
Removed, thanks